src/pages/messages/TasksPage.tsx

```diff
@@ -222,13 +229,33 @@ const TasksPage = () => {
               accordionPath={UrlPathTemplate.TasksNviCorrectionList}
               dataTestId={dataTestId.tasksPage.correctionList.correctionListAccordion}>
               <NavigationList>
-                <StyledSearchModeButton
-                  sx={{ mx: '1rem', mb: '1rem' }}
-                  data-testid={dataTestId.tasksPage.correctionList.correctionListRadioButton}
-                  isSelected={isOnCorrectionListPage}
-                  startIcon={<RadioButtonCheckedIcon />}>
-                  {t('tasks.correction_list')}
-                </StyledSearchModeButton>
+                <SelectableButton
```

Comment:
Alt dette burde flyttes inn i `NviMenuContent`(?). Ev. kunne dette vært tatt ut i en egen komponent, siden det er to ulike menyelementer